docs/source/admin_guide/backup.md

```diff
@@ -160,7 +180,50 @@ gsutil cp gs://<your_bucket_name>/backups/2021-04-23.tar .
 ### Digital Ocean
 
 Similar instructions, but use Digital Ocean spaces. This guide explains installation of the command-line tool:
-https://www.digitalocean.com/community/tutorials/how-to-migrate-from-amazon-s3-to-digitalocean-spaces-with-rclone
+<https://www.digitalocean.com/community/tutorials/how-to-migrate-from-amazon-s3-to-digitalocean-spaces-with-rclone>
+
+
+## JupyterHub Database
+
+The JupyterHub database will mostly be recreated whenever you start a new cluster, but should be backed up to save Dashboard configurations.
+
+You want to do something very similar to the NFS backup, above - this time you need to back up just one file, and it's on the PersistentVolume `hub-db-dir`.
+
+First, you might think you can just make a new `pod.yaml` file, this time specifying `claimName: "hub-db-dir"` instead of `claimName: "jupyterhub-dev-share"`. However, `hub-db-dir` is 'Read Write Once' - the 'Once' meaning it can only be mounted to one pod at a time, but the JupyterHub pod will already have this mounted! So the same approach will not work here.
```

Comment:
```{suggestion}
It is important to note that `hub-db-dir` is 'Read Write Once' - the 'Once' meaning it can only be mounted to one pod at a time. in this case the `JupyterHub pod`. Instead of creating a new `pod.yml` file you will need to follow the next steps:

```

Reply:
I appreciate we jump into 'story telling mode' here, but I think I should leave it as anyone genuinely trying to understand will need to get to the bottom of this. As you say, everyone else is just copying and pasting anyway!